src/amber/cli/recipes/filters.cr

```diff
@@ -0,0 +1,54 @@
+# add a pluralize filter to liquid
+# this should be a temporary measure until pluralize is available as a filter
+# in the liquid shard
```

Comment:
This comment isn't correct as this file is adding pluralize as well as underscore. It's probably redundant anyway.